orc8r/gateway/python/magma/common/redis/containers.py

```diff
@@ -13,9 +13,9 @@
 from copy import deepcopy
 import redis
 from redis.lock import Lock
+import redis_lock
```

Comment:
one minor concern, is the new dependency we are adding here. I saw that redis python library has a lock extension method as well, perhaps we can create a small lock wrapper with context library and a thread to periodically renew the lock. wdyt ?

Reply:
@karthiksubraveti 
From what I understand, the lock extension logic requires a small lua script. 
(You can actually see how the library does it here: https://github.com/ionelmc/python-redis-lock/blob/master/src/redis_lock/__init__.py )
It's a small enough library that if we want this feature, we'll just re-implement more or less the same thing. 
But I will look into the exact cost of a new dependency, just so we have the full picture.